config/scalable-server-vars.yml

```diff
@@ -0,0 +1,15 @@
+---
+# Copyright (c) Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT license. See LICENSE file on the project webpage for details.
+
+# Sample Open edX on Azure configuration
+# see: https://github.com/edx/configuration/blob/master/playbooks/roles/edxapp/defaults/main.yml
+
+# Database configuration for scalable ARM template

```

Comment:
Wondering if scalable.yml would be better file name, since "server-vars" is default for edx configurations.